plane/source/docs/automatic-landing.rst

```diff
@@ -4,52 +4,79 @@
 Automatic Landing
 =================
 
-This article explains how to land Plane as part of a mission plan and includes information about how a landing can be safely aborted.
+This article explains how to autonomously land Plane as part of a mission plan and includes information about how a landing can be safely aborted.
 
 Configuring for Automatic Landing
 =================================
 
-Plane can automatically land an aircraft, as part of a mission plan.
+To land the plane you need to add a :ref:`NAV_LAND <mav_cmd_nav_land>` command to the end of your mission indicating the latitude, longitude and altitude of your desired touchdown point. When the preceeding waypoint is reached, it will steer to the touchdown point.
+In most cases, the altitude should be set to 0. During landing, the autopilot will shut down the throttle and hold the current heading when the plane reaches the flare point, controlled by the parameters described below.
 
-To land the plane you need to add a :ref:`NAV_LAND <mav_cmd_nav_land>` command to the end of your mission indicating the latitude, longitude and altitude of your desired touchdown point. 
-In most cases, the altitude should be set to 0. 
-During landing, the autopilot will shut down the throttle and hold the current heading when the plane reaches the flare point, controlled by the parameters described below.
+For many vehicles, most of the parameters associated with autolanding can be left at their defaults. The basic parameters are covered below in the Basic Autolanding Section for vehicles not using an airspeed sensor and/or rangefinder. For more advanced configuration, see the sections listed below:
 
-.. _automatic-landing_key_parameters:
+.. toctree::
+   :maxdepth: 1
 
-Key Parameters
---------------
+    Aborting an Autolanding <aborting-autolanding>
+    Precision Autolanding <precision-autolanding>
+    RangeFinder Autolanding <rangefinder-autolanding>
+    Using Reverse Thrust in Autolanding <reverse-thrust-autolanding>
+    Using DO_LAND_START <do-land-start>
+
+Basic Autolanding
+=================
+
+The phases of an autolanding are:
+
+- Passing final approach waypoint
+- Navigating to the flare point on a "glide-slope" (ie controlled descent)
+- Flare and touchdown
+
+.. image:: ../../../images/autolanding.jpg
+   :target: ../../_images/autolanding.jpg
+
+Setting Up the Approach Waypoint
+--------------------------------
+
+The autolanding begins after reaching the last navigation waypoint. This waypoint must be far enough from the touchdown point and at an altitude that a reasonable glide slope can be obtained. Without an airspeed sensor, the speed will be at least that of normal waypoint navigation and usually higher, due to the descent. So having a glide slope (angle between last waypoint and touchdown altitude) that is not only obtainable within the TEC_SINK_MAX rate of the vehicle, but well below that rate to avoid overspeeding which makes adjusting the flare more difficult, is required.
```

Comment:
This last sentence here "So having a glide slope ..." is very complicated. It's difficult to understand what it's trying to say. What I _think_ it's trying to say is this:
It is important to have a glide slope that is obtainable with the TECS_SINK_MAX rate of the vehicle and well below that rate. This will avoid overspeeding and make it easier for the autopilot to adjust the flare. 

I think you can leave out the (definition of glide slope) here as your excellent diagram below explains it much better. 

Also - overspeeding needs to be defined. I still don't know what this really means.

Reply:
will modify

Reply:
Also typo - TECS_SINK_MAX (not TEC)